### PR TITLE
Audio Source Position

### PR DESCRIPTION
**Pull request by float3**

### CODE REVIEW

File: Packages/com.llealloo.audiolink/Runtime/Shaders/AudioLink.cginc [hunk 1]
```diff
@@ -310,4 +311,9 @@ uint AudioLinkGetCustomString2Char(uint charIndex)
     return AudioLinkGetGlobalStringChar(AUDIOLINK_STRING_CUSTOM2, charIndex);
 }
 
+// Returns the position of the AudioLink AudioSource.
+float3 AudioLinkGetAudioSourcePosition()
+{
+    return AudioLinkData(ALPASS_GENERALVU_SOURCE_POS);
```
Comment (pema99) [inline Packages/com.llealloo.audiolink/Runtime/Shaders/AudioLink.cginc:317]: pls no implicit vector truncation

**Comment by pema99** [comment]
other than that it looks fine

**Comment by pema99** [comment]
Could you add the relevant info to the docs?
The new define in the list of defines, and info the section about general vu

File: Packages/com.llealloo.audiolink/Runtime/Shaders/AudioLink.cginc (hunk 1 above)
Comment (float3) [reply Packages/com.llealloo.audiolink/Runtime/Shaders/AudioLink.cginc]: 44409bf

**Comment by float3** [comment]
do not merge before #206

**Comment by pema99** [comment]
In the docs, don't say absolute position, say position in worldspace

**Comment by pema99** [comment]
Other than that looks good

**Review by pema99 — APPROVED** [review]

**Pull request merged**